nakamoto-release-tsd.md

```diff
@@ -0,0 +1,973 @@
+# Nakamoto Release | Technical Specification Document
+
+# Executive Summary:
+
+>👉 *At a high-level what is The Nakamoto Release - its purpose, target audience, and benefits?*
+</br>
+
+The Nakamoto Release proposes a new consensus protocol for Stacks blockchain, enhancing security and significantly reducing transaction confirmation latency to seconds.
+
+This document describes a new consensus protocol for the Stacks blockchain, The Nakamoto Release, superseding the competitive mining protocol described in SIP-001 and SIP-007. In this proposed system, both miners and PoX stackers _cooperate_ through Byzantine fault-tolerant agreement to produce a linearized stream of confirmed transactions, in which the materialized view of the chainstate is periodically recorded to the Bitcoin blockchain. 
+
+This new arrangement allows for a significantly lower transaction confirmation latency time (*on the order of seconds instead of hours*), while also increasing the security budget of the chain from 51% of the miners' Bitcoin spends to _at least_ the sum of 67% of the miners' Bitcoin spends plus 67% of the total STX stacked. The chain's security budget reaches 51% of Bitcoin's mining power through periodic chainstate snapshots written to the Bitcoin chain by block producers and stackers, which are confirmed by the Bitcoin network as regular transactions. Once a Stacks chainstate snapshot is confirmed by the Bitcoin network, no subsequent Stacks fork can revert it -- any Stacks chain reorganization requires a Bitcoin chain reorganization.
+
+## In summary, The Nakamoto Release:
+
+- Replaces competitive mining (ratified in SIP-001 and SIP-00 with a cooperative system between miners and PoX stackers through Byzantine fault-tolerant (BFT) agreement that produces a linearized stream of confirmed transactions with periodic chainstate recording to Bitcoin blockchain.
```

Comment:
Number missing here in second SIP

Reply:
https://github.com/stacks-network/stacks/pull/485